### PR TITLE
Update the docs for hitTest()

### PR DESCRIPTION
**Pull request by Hixie**
cc @abarth

### CODE REVIEW

File: packages/flutter/lib/src/rendering/object.dart [hunk 1]
```diff
@@ -2130,10 +2130,7 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
   // following (with the signature being whatever passes for coordinates
   // for this particular class):
   // bool hitTest(HitTestResult result, { Point position }) {
-  //   // If (x,y) is not inside this node, then return false. (You
-  //   // can assume that the given coordinate is inside your
-  //   // dimensions. You only need to check this if you're an
-  //   // irregular shape, e.g. if you have a hole.)
+  //   // If (x,y) is not inside this node, then return false.
```
Comment (HansMuller) [inline packages/flutter/lib/src/rendering/object.dart]: Maybe [position] instead of (x,y)

**Comment by HansMuller** [comment]
LGTM

File: packages/flutter/lib/src/rendering/object.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/rendering/object.dart]: "You must not add yourself to /result/ if you return false"

^^ Is this part still accurate?

Comment (Hixie) [inline packages/flutter/lib/src/rendering/object.dart]: updated this part too.

**Comment by Hixie** [comment]
I applied even more changes. Will check in when green unless there's more comments by then.

**Pull request merged**